src/libtorch.cc

```diff
@@ -1532,9 +1527,7 @@ SetStringInputTensor(
                   std::to_string(element_idx + 1) + " for inference input '" +
                   name + "', expecting " + std::to_string(request_element_cnt))
                   .c_str()));
-      FillStringTensor(
-          input_list, tensor_offset + element_idx,
```

Comment:
`tensor_offset` is no longer needed? If it becomes unused parameter, then you can remove it from the function signature and propagate the change up on the call stack.

Reply:
Thanks for the catch. Removed the unused variable.